File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> minheap= new PriorityQueue<>();
        int ans; 
        for(int num: nums){
            minheap.offer(num);
            while(minheap.size()>k){
                minheap.poll();
            }
        }
        ans= minheap.poll();
        return ans;
    }
}

File: Data Structures & Algorithms/pow-x-n/submission-0.cpp
class Solution {
public:
    double myPow(double x, int n) {
        double ans=0; 
        if(n==0){
            return 1.0;
        }
        ans= myPow(x,n/2);
        if(n%2==0){
            return ans*ans;
        }
        else {
            if(n>0){
                return ans*ans*x;

            }
            else {
                return ans*ans/x;
            }
        }
    }
};
